File: WebViewApp/app/src/main/java/com/example/webviewapp/contract/LabelContract.java
package com.example.webviewapp.contract;

import com.example.webviewapp.data.Record;

import java.util.List;

public interface LabelContract {
    interface View {
        void setEditTextVisibility(Boolean isVisible);
    }

    interface Presenter {
        List<Record> getData();

        void checkScrolled(int dy);

        List<Record> refreshRecord();
    }
}

File: WebViewApp/app/src/main/java/com/example/webviewapp/contract/LabelContract.kt
package com.example.webviewapp.contract

import com.example.webviewapp.data.Record

interface LabelContract {
    interface View {
        fun setEditTextVisibility(isVisible: Boolean?)
    }

    interface Presenter {
        val data: List<Record?>?
        fun checkScrolled(dy: Int)
        fun refreshRecord(): List<Record?>?
        fun deleteLabel(pri: List<String?>?)
    }
}